seqr/views/utils/anvil_metadata_utils.py

```diff
@@ -411,9 +411,18 @@ def _get_transcript_field(field, config, transcript):
 def _get_subject_row(individual, has_dbgap_submission, airtable_metadata, individual_ids_map, get_additional_individual_fields, format_id):
     paternal_ids = individual_ids_map.get(individual.father_id, ('', ''))
     maternal_ids = individual_ids_map.get(individual.mother_id, ('', ''))
+    sex = individual.sex
+    sex_detail = None
+    if sex in Individual.MALE_ANEUPLOIDIES:
+        sex_detail = sex
+        sex = Individual.SEX_MALE
+    elif sex in Individual.FEMALE_ANEUPLOIDIES:
+        sex_detail = sex
+        sex = Individual.SEX_FEMALE
     subject_row = {
         'participant_id': format_id(individual.individual_id),
-        'sex': Individual.SEX_LOOKUP[individual.sex],
+        'sex': Individual.SEX_LOOKUP[sex],
```

Comment:
Is the lookup within `SEX_LOOKUP` necessary here if `sex` has already been reassigned to either `SEX_MALE` or `SEX_FEMALE` in the above `if`/`else` block?

Reply:
is there a need to do the reassignment then, since Individual.SEX_LOOKUP contains the aneuploidies now?

Reply:
the lookup  value for an aneuploidy is the aneuploidy itself, which is an invalid sex value for the reports we generate, which is what this code is for, but everywhere else that is the desired lookup behavior so its better to have this one special casing in our reporting code for aneuploidies than need to add complicated logic everywhere else sex is referenced

Reply:
ah... ok.  I think I was getting confused by the `SEX_LOOKUP` on `MatchmakerSubmission` being present on this diff, where the lookup of the aneuploidy is not the aneuploidy, but `Individual.SEX_LOOKUP` being unchanged.  

A comment about this being a special case would be helpful!